src/timer/stopwatch.rs

```diff
@@ -65,12 +65,7 @@ macro_rules! stopwatches {
                 }
 
                 pub fn now(&self) -> Instant {
-                    let _high = 0;
-                    $(
-                        let _high = self.tim.cnt.read().$cnt_h().bits() as u32;
-                    )*
-                    let low = self.tim.cnt.read().$cnt().bits() as u32;
-                    Instant(low | (_high << 16))
+                    Instant(self.tim.cnt.read().bits())
```

Comment:
@dotcypress is there a reason this wasn't done this way originally?

Reply:
Hmm, I don't recall why I did this.